src/frontend/js/widgets/SyllabusCourseRunsList/components/CourseProductItem/components/CourseProductCourseRuns/EnrolledCourseRun.tsx

```diff
@@ -91,7 +94,7 @@ const EnrolledCourseRun = ({ enrollment }: Props) => {
           {isStarded && (
             <Button
               size="small"
-              href={enrollment.course_run.resource_link}
+              href={`/dashboard${generatePath(LearnerDashboardPaths.ORDER, { orderId })}`}
```

Comment:
I think you should use the same logic than here.
https://github.com/openfun/richie/blob/c32b64e742fffd56acc2817f3fd271ed205025ad/src/frontend/js/components/SaleTunnel/components/SaleTunnelStepResume/index.tsx#L74-L77